docs/metadata.rst

```diff
@@ -381,6 +382,12 @@ of ``RE.md`` can persist between sessions.
     RE.md = StoredDict('some/path/here')  # path is a file
 
 .. rubric:: PersistentDict
+
+Bluesky formerly recommended using :class:`~bluesky.utils.PersistentDict` --- a
+Python dictionary backed by a zict directory. This approach proved
```

Comment:
If we are going to provide this commentary it should be correct. 

The issue is that if multiple instances `PersistentDict` work on the same directory (either in one or multiple processes) they would get out of sync and look for files that did not exist.

Reply:
See commit 63228e49ff